### PR TITLE
Fix: LosingPositionsRatioCriterion#betterThan

### PR DESCRIPTION
**Pull request by nimo23**
Changes proposed in this pull request:
- fixes `LosingPositionsRatioCriterion#betterThan()`
- fixes ﻿`LosingPositionsRatioCriterionTest`
- added javadoc for `betterThan()` methods of all criterions

- [x] added an entry with related ticket number(s) to the unreleased section of `CHANGES.md`

### CODE REVIEW

File: ta4j-core/src/main/java/org/ta4j/core/BacktestExecutor.java [hunk 1]
```diff
@@ -23,15 +23,15 @@
  */
 package org.ta4j.core;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import org.ta4j.core.cost.CostModel;
```
Comment (team172011) [inline ta4j-core/src/main/java/org/ta4j/core/BacktestExecutor.java]: Changes not related and not relevant

File: ta4j-examples/src/main/java/ta4jexamples/backtesting/SimpleMovingAverageRangeBacktest.java [hunk 2]
```diff
@@ -35,10 +44,8 @@
 import org.ta4j.core.reports.TradingStatement;
 import org.ta4j.core.rules.OverIndicatorRule;
 import org.ta4j.core.rules.UnderIndicatorRule;
-import ta4jexamples.loaders.CsvBarsLoader;
 
-import java.util.ArrayList;
-import java.util.List;
+import ta4jexamples.loaders.CsvBarsLoader;
```
Comment (team172011) [inline ta4j-examples/src/main/java/ta4jexamples/backtesting/SimpleMovingAverageRangeBacktest.java]: Changes are not related and not relevant

File: ta4j-examples/src/main/java/ta4jexamples/loaders/JsonBarsSerializer.java [hunk 3]
```diff
@@ -23,19 +23,20 @@
...
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
+import org.ta4j.core.BarSeries;
+
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 
+import ta4jexamples.loaders.jsonhelper.GsonBarSeries;
+
```
Comment (team172011) [inline ta4j-examples/src/main/java/ta4jexamples/loaders/JsonBarsSerializer.java]: Changes not related and not relevant

File: ta4j-examples/src/main/java/ta4jexamples/loaders/jsonhelper/GsonBarData.java [hunk 4]
```diff
@@ -23,13 +23,13 @@
...
-import org.ta4j.core.Bar;
-import org.ta4j.core.BaseBarSeries;
-
 import java.time.Instant;
 import java.time.ZoneId;
 import java.time.ZonedDateTime;
 
+import org.ta4j.core.Bar;
+import org.ta4j.core.BaseBarSeries;
+
```
Comment (team172011) [inline ta4j-examples/src/main/java/ta4jexamples/loaders/jsonhelper/GsonBarData.java]: Changes not related and not relevant

File: ta4j-examples/src/main/java/ta4jexamples/loaders/jsonhelper/GsonBarSeries.java [hunk 5]
```diff
@@ -23,14 +23,14 @@
  */
 package ta4jexamples.loaders.jsonhelper;
 
+import java.util.LinkedList;
+import java.util.List;
+
```
Comment (team172011) [inline ta4j-examples/src/main/java/ta4jexamples/loaders/jsonhelper/GsonBarSeries.java]: Changes not related and not relevant

File: ta4j-examples/src/test/java/ta4jexamples/loaders/JsonBarsSerializerTest.java [hunk 6]
```diff
@@ -23,18 +23,20 @@
...
 import java.io.IOException;
 import java.math.BigDecimal;
 import java.util.concurrent.ThreadLocalRandom;
 
-import static org.junit.Assert.*;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.ta4j.core.Bar;
+import org.ta4j.core.BarSeries;
```
Comment (team172011) [inline ta4j-examples/src/test/java/ta4jexamples/loaders/JsonBarsSerializerTest.java]: Changes not related and not relevant

**Review by team172011 — COMMENTED** [review]
Please undo not related changes

File: ta4j-examples/src/test/java/ta4jexamples/loaders/JsonBarsSerializerTest.java (hunk 6 above)
Comment (nimo23) [reply ta4j-examples/src/test/java/ta4jexamples/loaders/JsonBarsSerializerTest.java]: @team172011  Are you really forcing me to change each file only because you can or won't not merge those simple things? I will not loose my time..so feel free to close this issue without merging.

File: ta4j-examples/src/main/java/ta4jexamples/backtesting/SimpleMovingAverageRangeBacktest.java (hunk 2 above)
Comment (nimo23) [reply ta4j-examples/src/main/java/ta4jexamples/backtesting/SimpleMovingAverageRangeBacktest.java]: Those imports are **unnessary**..and instead of saying "Thank you", you reprove me in a unreasonable way. I will not change it. So feel free to close this issue without merging and leave this lib in such a messy way.

**Comment by team172011** [comment]
> Those imports are unnessary..and instead of saying "Thank you", you reprove me in a unreasonable way. I will not change it. So feel free to close this issue without merging and leave this lib in such a messy way.

> @team172011 Are you really forcing me to change each file only because you can or won't not merge those simple things? I will not loose my time..so feel free to close this issue without merging.

Please don't understand me wrong, @nimo23 I really appreciate your contributions!
But I also uphold my view that we should not change the import order in a lot of unrelated classes and you do so in a lot of PRs. Thats why I decided to point to it with my comments again. Pull Requests and the review function are intended for discussing and commenting things like this. 

It is not much effort to undo those changes. You can just do a git checkout develop for the corresponding classes like:
```
git checkout develop ta4j-examples/src/test/java/ta4jexamples/loaders/JsonBarsSerializerTest.java ta4j-examples/src/main/java/ta4jexamples/loaders/jsonhelper/GsonBarSeries.java ta4j-examples/src/main/java/ta4jexamples/loaders/jsonhelper/GsonBarData.java
```

And I see you already did this, thanks!
